pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
We should never change sccs to add privileges.  That changes the meaning of
the scc and can lead to security breaches.  If it has to run priviliged,
add it to privileged.

On Nov 9, 2015, at 11:56 AM, Mark Turansky <notifications@github.com> wrote:

In pkg/cmd/server/kubernetes/master.go
<https://github.com/openshift/origin/pull/5792#discussion_r44322502>:

> @@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
>  }
>
>  func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
> +	privilged := true

I am confused. What do we want to do with the service account?

Hostmount was added for the recycler (though maybe not exclusively).
Requiring the user to add Privileged to the Houstmount SCC seems logical,
no? They would explicitly opt-in to enable this feature.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5792/files#r44322502>.